integrations/gen_docs_integrations.py

```diff
@@ -25,6 +25,9 @@ def cleanup():
     for element in Path("integrations/cloud-notifications").glob('**/*/'):
         if "integrations" in str(element) and not "metadata.yaml" in str(element):
             shutil.rmtree(element)
+    for element in Path("integrations/logs").glob('**/*/'):
+        if "integrations" in str(element) and "metadata.yaml" not in str(element)::
```

Comment:
```suggestion
        if "integrations" in str(element) and "metadata.yaml" not in str(element):
```

Reply:
wow, probably mistype after I ran it, because I tested the output md